### PR TITLE
tests: merge and report coverage

### PR DESCRIPTION
**Pull request by LukasHirt**
Merge test coverage from unit and integration tests

### CODE REVIEW

**Review by kulmann — DISMISSED** [review]
nice, thanks for taking care! 😍

File: .drone.star [hunk 1]
```diff
@@ -1026,7 +1026,16 @@ def unitTests(ctx):
...
+                             "yarn test:integration --coverage",
+                         ],
+                     },
+                     {
+                         "name": "report-coverage",
+                         "image": "owncloudci/nodejs:16",
+                         "pull": "always",
+                         "commands": [
+                             "yarn test:coverage:merge",
+                             "yarn test:coverage:report"
```
Comment (kulmann) [inline .drone.star:1038]: Needs a trailing comma

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18288/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by LukasHirt** [comment]
Hmm, no coverage information... I guess something is missing now

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18288/12/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by LukasHirt** [comment]
Postponing until SonarCloud is working fully again...

**Comment by JammingBen** [comment]
Closing as we don't have integration tests anymore.

**Pull request closed without merging**